spec/oauth2/access_token_spec.rb

```diff
@@ -44,6 +44,13 @@ def assert_initialized_token(target) # rubocop:disable Metrics/AbcSize
       assert_initialized_token(target)
     end
 
+    it 'does not modify opts hash' do
+      hash = {:access_token => token, :expires_at => Time.now.to_i + 200, 'foo' => 'bar'}
+      hash_before = hash.dup
+      AccessToken.from_hash(client, hash)
+      expect(hash).to eq(hash_before)
+    end
+
```

Comment:
I'm wrong here, too: `AccessToken.from_hash` deletes the access token, so this does show something relevant. The `'foo' => 'bar'` is probably not relevant. The `:expires_at => Time.now.to_i+200` shows that `#initialize` doesn't modify it, though it could be a red-herring to have the value look so intentionally chosen.